### ISSUE TITLE
Add vue-async-computed-decorator to README.md

### ISSUE THREAD

**Issue by nwtgck** [foxbenjaminfox/vue-async-computed#90]
Hi @foxbenjaminfox.

I think the decorator package, the like of `vue-async-computed-decorator` should be on README of this repository.

**Comment by foxbenjaminfox** [comment]
I agree, but for now I'm treating `vue-async-computed-decorator` as something of a beta release. I want to give it a few days or maybe a week or two to iron out all the details (such as  [this bug](https://github.com/foxbenjaminfox/vue-async-computed-decorator/pull/1) that just now got fixed), and after I'm sure that things are more or less stable I'll then release an official `v1.0.0` of `vue-async-computed-decorator` and link to it from here.

**Comment by nwtgck** [comment]
@foxbenjaminfox Thanks. I understand the plan. I'll use the decorator package.
